moveit_ros/planning/moveit_cpp/src/moveit_cpp.cpp

```diff
@@ -245,6 +245,10 @@ MoveItCpp::execute(const robot_trajectory::RobotTrajectoryPtr& robot_trajectory,
   // Execute trajectory
   moveit_msgs::msg::RobotTrajectory robot_trajectory_msg;
   robot_trajectory->getRobotTrajectoryMsg(robot_trajectory_msg);
+  trajectory_execution_manager_->push(robot_trajectory_msg, controllers);
+  trajectory_execution_manager_->execute();
```

Comment:
These changes are not related to this PR

Reply:
These were part of the function before the `blocking` parameter was removed [line 226](https://github.com/ros-planning/moveit2/blob/main/moveit_ros/planning/moveit_cpp/src/moveit_cpp.cpp) I don't know why it shows these are being new changes.